src/components/lyne-cta-button/lyne-cta-button.stories.js

```diff
@@ -1,12 +1,33 @@
 import events from './lyne-cta-button.events.ts';
 import { h } from 'jsx-dom';
 
+const types = {
+  control: {
+    type: 'radio'
```

Comment:
See comments below regarding the naming. Maybe variant is not the right fit either, because the actuall variants could be "Text with icon", "Text only", "Icon only", which again could come in all visual characteristics (primary, secondary, etc).

Reply:
I already changed it here, please pull latest

Reply:
As mentioned, let's try to define the component-internal variants "Text with icon", "Text only", "Icon only" via the attributes for `label` (existing) and `icon` (not yet implemented)